### DISCUSSION TITLE
zh/docs/admin-portal

### DISCUSSION THREAD

**Discussion by giscus** [casbin/casbin#1131] [Docs comments]
# zh/docs/admin-portal

Casbin 管理门户网站

https://casbin.io/zh/docs/admin-portal

**Comment by wushifeng** [comment]
Where does the PML Model Editor above come from?

**Reply by hsluoyz** [reply]
Casbin editor: https://editor.casbin.org/

source code: https://github.com/casbin/casbin-editor
